packages/core/src/components/modal/modal.tsx

```diff
@@ -46,6 +46,9 @@ export class TdsModal {
    * will be decided by this prop and will need to be controlled from the outside. */
   @Prop() show: boolean;
 
+  /** Enables the close button. */
```

Comment:
```suggestion
  /** Shows or hides the close [X] button. */
```
Just to be more specific. Enable might mean that button is there but we do something with preventDefault etc...